src/common-declarations.js

```diff
@@ -53,8 +53,19 @@ const commonDeclarations = {
   paddingBottom: [ 0 ],
   paddingLeft: [ 0 ],
 
+  minWidth: [ '100%' ],
   maxWidth: [ '100%' ],
-  height: [ 'auto' ],
+  width: [
+    'auto',
+    0,
+    '50%',
```

Comment:
Starting to get into a subjective area where it isn't questionable on being "common". I'd be fine with everything except `50%`.